GoogleForCreators/web-stories-wp
# Settings: Add additional helpful messaging for AdSense

## Context



## Summary

Update wording on adsense text in dashboard settings page.

## Relevant Technical Choices



## To-do



## User-facing changes

Update wording in settings screen. 

<img width="728" alt="Screenshot 2021-05-12 at 16 18 10" src="https://user-images.githubusercontent.com/237508/118000434-b66e7000-b33d-11eb-992c-9429799f05c8.png">

## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Go the settings screen
2. Verify the help text for AdSense has been updated
3. Verify that all links work in said text

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6782
